src/App.js

```diff
@@ -1,14 +1,19 @@
 import './App.css';
 import { Header } from './components/Header';
 import { Main } from './components/Main';
+import {Routes, Route} from 'react-router-dom';
 
 function App() {
   return (
-    <div className="App">
-      <Header/>
-      <Main/>
-    </div>
-  );
+		<div className='App'>
+			<Header />
+			<Routes>
+				<Route path='/' element={<Main />} />
+				<Route path='/reviews' element={<Main />} />
+				<Route path='/reviews/:review_id' element={<Main />} />
+			</Routes>
+		</div>
+	);
```

Comment:
in your routes you are just rendering main which is rendering `TopBar` and `Content`  you should move `TopBar` to below `Header` if you want it rendering all the time and in your Routes then render either `Single-review` or `Review-list` 
using `Content` currently is just doing the job of react-router-dom which is unnecessary coding for yourself